EIPS/eip-3238.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 3238
+title: Vatnajoekull Difficulty Bomb Delay
+author: Afri Schoedon (@q9f)
+discussions-to: https://github.com/ethereum/EIPs/pull/3238
+type: Standards Track
+category: Core
+status: Draft
+created: 2021-01-25
+---
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") and slowly accelerating. This EIP proposes to delay the difficulty bomb for approximately ten months with the Vatnajoekull fork, a yet to be scheduled adjustment.
+
+## Abstract
+Starting with `VTNJKLL_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting to the client that the difficulty bomb is adjusting eleven million blocks later than previously specified with the Homestead fork; that is two million blocks later than specified with the latest delay as included in the Muir Glacier fork ([EIP-2387](./eip-2387.md)).
```

Comment:
As mentioned above, EIPs are unrelated to hardfork coordination.  While you are free to choose any variable names you want, I recommend something simpler and more clear given the lack of context of hard fork naming in this repository like `FORK_BLOCK_NUMBER`.  (on a personal note, I also recommend spelling out words since you'll write this document once and it will be read thousands of times)